### PR TITLE
Speakers, Volunteers and website updates

### PR DESCRIPTION
**Pull request by swapnilogale**
@ericholscher @plaindocs Just set up a page for announcing speakers (I've used plain text instead of a yaml, but can change it once I have time to upload the updated speaker abstracts!)

### CODE REVIEW

**Comment by swapnilogale** [comment]
@plaindocs - is there something wrong with Netlify preview?

**Comment by plaindocs** [comment]
@swapnilogale what do you mean? 

https://deploy-preview-1075--dentist-canary-64055.netlify.com/conf/australia/2019/

looks ok to me

**Comment by swapnilogale** [comment]
@plaindocs - yup, all good now. Once you get the speakers into the YAML, I reckon we can ship this

**Comment by ericholscher** [comment]
@swapnilogale what needs to be done beyond https://deploy-preview-1075--dentist-canary-64055.netlify.com/conf/australia/2019/speakers/ ?

**Comment by swapnilogale** [comment]
@ericholscher missing a few speakers. I have updated the AU spreadsheet with these details.

**Comment by swapnilogale** [comment]
@ericholscher @plaindocs - I am happy for this to go out - I've added all speaker info we need for this announcement. Can you review please?

File: docs/_data/2019.australia.speakers.yaml [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+  - name: Alec Clews
+    slug: alec-clews
+    twitter: '@alecthegeek'
+    website: ''
+  title: Creating API documentation for international communities
+- abstract: '<p>"What specific information architecture issues do you face when creating documentation?
+    Learn how core IA principles can be adapted for documentation.</p>'
+  slug: information-architecture-for-documentation-elle-geraghty
+  speakers:
+  - name: Elle Geragthy
```
Comment (plaindocs) [inline docs/_data/2019.australia.speakers.yaml]: ellegeraghty I think

Comment (swapnilogale) [reply docs/_data/2019.australia.speakers.yaml]: Resolved

**Comment by swapnilogale** [comment]
All good to go @plaindocs

**Review by plaindocs — APPROVED** [review]
* `docs/_static/img/speakers/elle-geraghty - Copy.jpg` can probably be deleted
* some other speaker pics might need checking or we just need to add a few more
* Monica Kiss needs a newline to get the numbered list working or somethin

Otherwise we can merge this and iterate.

**Comment by swapnilogale** [comment]
Let's merge and I'll fix this in a couple of weeks when we publish the
schedule.

Swapnil

**Pull request merged**